plugins/monitoring/app/views/monitoring/alarm_definitions/create_expression.html.haml

```diff
@@ -91,7 +91,7 @@
           = link_to "", '#', id: 'reload_statistic_btn', onclick:'show_statistic()', class: 'fa fa-refresh reload-statistic-btn', title: 'reload statistic'
           %svg#statistic{style:"width:425px; height:370px;"}
           %span.help-block
-            The graph shows the historical data from the last 2 hours. Playing with the dimensions, statistical function and period should help you to find a good treshold value.
+            The graph shows the historical measuring points from the last 2 hours. Playing with the dimensions, statistical function and period should help you to find a good treshold value.
```

Comment:
"Measuring points" sounds weird. What about "data points"? Also, the last 2 hours are hardly "historical". :)